src/blocks/BoardListBlock.tsx

```diff
@@ -72,17 +292,28 @@ export interface EmptyProps {
 }
 export interface ItemProps {
   slug: string;
-  description?: string;
+  color: string;
+  avatar: string;
+  link?: LinkWithAction;
+  description: string;
   _onSelected?: (slug: string) => void;
   _selected?: boolean;
   options: { name: string; link: LinkWithAction }[];
+  muted?: boolean;
+  updates?: number | boolean;
+  outdated?: boolean;
+  _onLinkedClicked?: () => void;
+  _onPinned?: () => void;
+  _onMuted?: () => void;
 }
 
 export interface BoardListBlockProps {
-  icon?: string;
+  icon: string | IconProp;
```

Comment:
If title is optional I'd make icon optional too. I'd say let's just make both required.